src/main/java/com/iemr/tm/repo/user/UserLoginRepo.java

```diff
@@ -0,0 +1,17 @@
+package com.iemr.tm.repo.user;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import com.iemr.tm.data.login.Users;
+
+
+@Repository
+public interface UserLoginRepo extends CrudRepository<Users, Long> {
+
+	@Query(" SELECT u FROM Users u WHERE u.userID = :userID AND u.deleted = false ")
+	public Users getUserByUserID(@Param("userID") Long userID);
```

Comment:
For every request that comes though there will be SQL query against a table. Why do this?
This data should be ideally cached in Redis. If not present in Redis, make the query.

@indraniBan

Reply:
instead of redis changed to inbuilt query

Reply:
@indraniBan sorry, what does inbuilt query mean?

Reply:
@drtechie implemented redis concept, fetching from redis. but need to do one round testing from UI, if working fine will commit the code